tests/CMakeLists.txt

```diff
@@ -19,6 +19,11 @@ if(${CMAKE_CXX_COMPILER_ID} MATCHES "GNU|Clang")
         target_compile_options(lexy_test_base INTERFACE -O0 -g --coverage)
         target_link_options(lexy_test_base INTERFACE --coverage)
     endif()
```

Comment:
Nit: please remove the endif() above.

Reply:
I'm not sure what you mean by this, could you elaborate?

Reply:
You currently have `if() ... endif() elseif() ..  endif()`. The middle endif () is not necessary.

Reply:
There's a nested if in there that asserts on clang-cl.

Reply:
... The github app didn't show indentation and the lines above. nevermind. :D